packages/state-transition/src/cache/epochCache.ts

```diff
@@ -824,6 +903,10 @@ export class EpochCache {
 
     this.effectiveBalanceIncrements[index] = Math.floor(effectiveBalance / EFFECTIVE_BALANCE_INCREMENT);
   }
+
+  isAfterEIP6110(): boolean {
```

Comment:
micro-optimization: could this be a property instead of a function? the return of this function is constant through the epoch, and would make the codepaths for pre-eip6110 cheaper.

Reply:
> micro-optimization: could this be a property instead of a function? the return of this function is constant through the epoch, and would make the codepaths for pre-eip6110 cheaper.

Eh, I think it was a property at first and turned into a class method later. The discussion about it was here: https://github.com/naviechan/lodestar/pull/1#discussion_r1330326976 . Maybe I am misunderstanding

Reply:
:rofl: touche.. Both arguments are valid. What do you think is best here, method or pre-computed property?

Reply:
> 🤣 touche.. Both arguments are valid. What do you think is best here, method or pre-computed property?

Hahaha let's leave it as is